jekyll/_cci2/pipelines.adoc

```diff
@@ -0,0 +1,138 @@
+NOTE: The information in this section on configuration sources and triggers applies to accounts integrated with the xref:github-apps-integration#[CircleCI GitHub App] or xref:gitlab-integration#[GitLab]. There are two GitHub integration types, either OAuth or through the GitHub App. To find out which GitHub integration type you have refer to the xref:github-apps-integration#[GitHub App integration] page.
+
+#Add some information about why people want to go and add these things. On the Multiple config files setup page for GitHub app we have: You can use multiple configuration files in a CircleCI project and set up separate triggers for each one. Each configuration triggers a separate pipeline on a trigger event. If your existing configuration file is getting too large to manage, consider splitting it into multiple files.#
+
+=== Configuration source
+
+Configuration sources show CircleCI where a pipeline configuration for your project is stored. A configuration source can be set up for use when _triggering_ a pipeline.
+
+A project can have one or more configuration sources. Configurations include, but are not limited to, a `.circleci/config.yml` file in your repository.
+
+When you create a project in CircleCI and connect it to a repository, a configuration source is set up for you automatically, and connected to a configuration file. The connected configuration file will be either a file you commit  yourself or one generated by CircleCI based on the code in your project.
+
+image::{{site.baseurl}}/assets/img/docs/project-settings-configuration.png[Configuration source]
+
+==== Add a configuration source
+
+TIP: If you created a project xref:create-project#[in the CircleCI web app] you will find a configuration source is already set up.
+
+image::{{site.baseurl}}/assets/img/docs/add-configuration-source.png[Add a configuration source]
+
+To add a configuration source, follow these steps:
+
+. In the CircleCI web app, click on **Projects** in the sidebar
+. Click on the ellipsis next to your project (icon:ellipsis-h[]) and then click **Project Settings**
+. Select **Configuration** in the sidebar
+. Click btn:[Add Configuration Source]
+. Complete the form fields and options:
+** Give your configuration source a descriptive name
+** Under Configuration Source select the platform that matches where your project is set up (for example, GitHub App)
++
+NOTE: For GitLab Self-managed, you can input any instance that you have previously set up in CircleCI. If you wish to set a different feature branch or repository from a self-managed instance as a new configuration source, you will first need to add a new connection via your xref:gitlab-integration#organization-settings-integrations[menu:Organization Settings[Integrations]]. In either case, you will also need to enter your personal access token again to authorize this connection.
+** Authorize your connection if this is not already showing with a icon:check-circle[]
+** Select your project repository
```

Comment:
Should we mention here that, due to a bug,  CircleCI actually always looks for the config file in the repository from which the TRIGGERING event is emitted? 
So users should not enter a different repository

Reply:
Just to confirm: 

* users can select a different repo from the one set up for the project they are working in here? 
* but they need to select the repo that they will select when setting up the trigger connected to this config source?
* If they don't will it just operate as if they selected the same repo? Rather than error?
* In a project, which is connected to a repo, you can set up a config source and trigger for a different repo, but it's just that the config source and trigger need to be connected to the _same_ repo as each other?

Reply:
Very helpful, thank you, @BeFunes!